packages/snaps-execution-environments/lavamoat/browserify/worker-pool/policy.json

```diff
@@ -45,10 +45,16 @@
       "packages": {
         "@metamask/rpc-errors": true,
         "@metamask/snaps-sdk": true,
+        "@metamask/snaps-utils>luxon": true,
         "@metamask/superstruct": true,
         "@metamask/utils": true
       }
     },
+    "@metamask/snaps-utils>luxon": {
```

Comment:
We should check if this adds significantly to bundle size

Reply:
469 KB versus 385 KB on main. Potentially worth seeing if we can find a way to not include it 🤔

Reply:
Main:
![image](https://github.com/user-attachments/assets/83f0e650-5fa1-4061-84aa-0d13a823735e)

This branch:
![image](https://github.com/user-attachments/assets/920adaa8-45e0-422a-980b-1e51e7a739d0)

Reply:
We only check the request arguments on the execution env. side, I think we can safely remove the other structs form the environment execution exports.

Reply:
Same for the others handlers though